_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -361,9 +361,9 @@ mfxDefaultAllocatorVAAPI::AllocFramesHW(
             }
             else
             {
-                int width32 = 32 * ((request->Info.Width + 31) >> 5);
-                int height32 = 32 * ((request->Info.Height + 31) >> 5);
-                codedbuf_size = static_cast<mfxU32>((width32 * height32) * 400LL / (16 * 16));
+                int aligned_width  = mfx::align_value<int>(request->Info.Width,  32);
+                int aligned_height = mfx::align_value<int>(request->Info.Height, 32);
+                codedbuf_size = static_cast<mfxU32>((aligned_width * aligned_height) * 400LL / (16 * 16));
```

Comment:
`400LL` - do we really need `long long` here?

Reply:
This question need more investigation. This code is some copypaste from driver code as I remember. Need just to check which result we expected here